components/cluster/command/check.go

```diff
@@ -103,9 +103,17 @@ conflict checks with other clusters`,
 				}
 			}
 
-			sshConnProps, err := cliutil.ReadIdentityFileOrPassword(opt.identityFile, opt.usePassword)
-			if err != nil {
-				return err
+			// natvie ssh has it's own logic to find the default identity_file
+			if (gOpt.NativeSSH || gOpt.SSHType == executor.SSHTypeSystem) && !utils.IsFlagSetByUser(cmd.Flags(), "identity_file") {
```

Comment:
You have rewrite the `gOpt.SSHType` to `SSHTypeSystem` at https://github.com/pingcap/tiup/pull/699/files#diff-d3feba8feea94e4717a701aa9439c1a1R119, I think we can simplify this code.
```go
if gOpt.SSHType == executor.SSHTypeSystem && !utils.IsFlagSetByUser(cmd.Flags(), "identity_file") {...}
```